test/images/image-util.sh

```diff
@@ -119,7 +119,7 @@ push() {
   if [[ -f ${IMAGE}/BASEIMAGE ]]; then
     archs=$(listArchs)
   else
-    archs=${!QEMUARCHS[@]}
+    archs=${!QEMUARCHS[*]}
```

Comment:
```
In ./test/images/image-util.sh line 122:
    archs=${!QEMUARCHS[@]}
          ^--------------^ SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```